rpa-builder/6.3/modules/ROOT/pages/toolbox-variable-handling-generate-and-modify-date-and-time.adoc

```diff
@@ -6,80 +6,126 @@ templates or arrange the available time components as required, e.g.
 the time stamp using the date and time format selected.
-* *Date* The _Generate date and time.Date_ variable provides the date.
-* *Time* The _Generate date and time.Time_ variable provides the time.
-* *Year* The _Generate date and time.Year_ variable provides the year.
-* *Month* The _Generate date and time.Month_ variable provides the month
+* *Date*
++
+The _Generate date and time.Date_ variable provides the date.
+* *Time* 
++
+The _Generate date and time.Time_ variable provides the time.
+* *Year* 
++
+The _Generate date and time.Year_ variable provides the year.
+* *Month* 
++
+The _Generate date and time.Month_ variable provides the month
 as a decimal number.
-* *Day* The _Generate date and time.Day_ variable provides the day as a
+* *Day* 
++
+The _Generate date and time.Day_ variable provides the day as a
 decimal number.
-* *Hour* The _Generate date and time.Hour_ variable provides the number
+* *Hour* 
++
+The _Generate date and time.Hour_ variable provides the number
 of hours.
-* *Minute* The _Generate date and time.Minute_ variable provides the
+* *Minute* 
++
+The _Generate date and time.Minute_ variable provides the
 number of minutes.
-* *Second* The _Generate date and time.Second_ variable provides the
+* *Second* 
++
+The _Generate date and time.Second_ variable provides the
 number of seconds.
 
 == Wizard
 
-* *Generation mode* Use this radio button to select which time stamp is
-used:
-** *Current date and time* Use the current system time.
-** *Custom date and time* Use the time selected in the *DateTime* field.
-* *DateTime* Use to specify a time to be used as the *Custom date and
-time*.
-* *DateTime format* Select a predefined format or create a custom format
-using the available time components. The _Generate date and
-time.Formatted_ variable is used to output the specified format. The
-following time components are available
-** *dd*: Day (the day is written out as dddd).
-** *MM*: Month (the month is written out as MMMM).
-** *yy*: Year (the year ist written out as yyyy)
-** *HH*: Hour
-** *mm*: Minute
-** *ss*: Second
-* *Add to date* Use the following fields to specify the time offset that
-should be applied to the time stamp. You can specify the required value
-either by using the selector buttons or by entering the value directly.
-Positive and negative numbers can be used. The default value is 0 in
-each case.
-** *Days* Number of days
-** *Hours* Number of hours
-** *Minutes* Number of minutes
-** *Seconds* Number of seconds
+The Wizard enables you to generate a timestamp based on the specified time variables, select the starting day of the week, and specify the output format and language. 
+
+* *DateTime settings* 
++
+Enables you to specify the date to generate. 
++
+Select a *Generation mode* and specify the time variables to generate the desired date output. The available generation modes are: 
+
+** *Current system date and time*: Uses the current system time as the timestamp.

```

Comment:
timestamp is two different ways here. Best to make it consistent (either time stamp or timestamp--both are correct but I think timestamp is more commonly used in our world)

Reply:
S/R throughout